app/assets/javascripts/modules/namespaces/components/visibility.js

```diff
@@ -59,7 +60,8 @@ export default {
       set(this, 'onGoingRequest', true);
       set(this, 'newVisibility', visibility);
 
-      NamespacesService.changeVisibility(this.namespace.id, { visibility }).then(() => {
+      params = { namespace: { visibility: visibility } };
```

Comment:
In ES6 there's a syntax sugar for object properties that it's useful when you have an object key property equal to the variable name. In this case you could simply use `{ visibility }` and the effect is the same.